spec.md

```diff
@@ -300,23 +319,23 @@ that contains both context and data).
   * MUST be unique within the scope of the producer
 
 ### event-time
-* Type: Timestamp per [RFC 3339](https://tools.ietf.org/html/rfc3339)
+* Type: `Timestamp` per [RFC 3339](https://tools.ietf.org/html/rfc3339)
 * Description: Timestamp of when the event happened.
 * Constraints:
   * OPTIONAL
   * If present, MUST adhere to the format specified in
     [RFC 3339](https://tools.ietf.org/html/rfc3339)
 
 ### schema-url
-* Type: URI per [RFC 3986](https://tools.ietf.org/html/rfc3986)
+* Type: `URI` per [RFC 3986](https://tools.ietf.org/html/rfc3986)
 * Description: A link to the schema that the `data` attribute adheres to.
 * Constraints:
   * OPTIONAL
   * If present, MUST adhere to the format specified in
     [RFC 3986](https://tools.ietf.org/html/rfc3986)
 
 ### content-type
-* Type: String per [RFC 2046](https://tools.ietf.org/html/rfc2046)
+* Type: `String` per [RFC 2046](https://tools.ietf.org/html/rfc2046)
```

Comment:
For consistency sake, should RFC reference be moved up to the `String` definition?

Reply:
This is a String. And it need to conform to RFC2046 in this instance. So this one is correct,